yadageschemas/utils.py

```diff
@@ -28,7 +28,7 @@ def schemabase_uri(schemadir):
     if schemadir == None:
         schemabase = 'file://'+os.path.abspath(default_schemadir)
     elif schemadir=='from-github':
-        schemabase = 'https://raw.githubusercontent.com/lukasheinrich/cap-schemas/master/schemas'
```

Comment:
I unfortunately do not know what the `schemas` directory in `https://github.com/lukasheinrich/cap-schemas/tree/master/schema` was given that this repo has been renamed to `https://github.com/yadage/yadage-schemas/` and the redirect fails as there is no `schemas/` directory. So this should be fixed.